test/passing/tests/crlf_to_crlf.ml.ref

```diff
@@ -1,4 +1,5 @@
-let _ = {|
+let _ =
+  {|

```

Comment:
The previous output was expected. We call this "docking" in other contextes.

Reply:
I'll restrict the change to the janestreet profile, then. It is surprising to me, though, that the property used to determine whether to "dock" is the number of characters in the string and not something that more closely relates to how it "looks", like whether the first line is empty, the number of lines, or the length of the longest line.

Reply:
It's also surprising to me. I tried various rules in https://github.com/ocaml-ppx/ocamlformat/pull/2448 without success, as the changes to existing code is always large and the rule is always arbitrary. Thanks for the ideas, I'll try to come back to this change later.